doc/guides/west/config.rst

```diff
@@ -164,6 +165,13 @@ extension commands (found in :file:`scripts/west_commands`).
 
    * - Option
      - Description
+   * - ``build.board``
+     - String. If given, this the board used by :ref:`west build
```

Comment:
Nice to have a default board! I remember thinking about this as well a while back.